d3rlpy/algos/qlearning/base.py

```diff
@@ -520,6 +525,8 @@ def fitter(
         # save hyperparameters
         save_config(self, logger)
 
+        logger.watch_model(0, 0, gradient_logging_steps, self)
```

Comment:
I don't like it but it is required for wandb watch. Without this line wand doesn't track the first epoch.

Reply:
Can be fixed via https://github.com/takuseno/d3rlpy/pull/424#discussion_r1798196290